llama_stack/providers/adapters/safety/together/together.py

```diff
@@ -3,12 +3,40 @@
 #
 # This source code is licensed under the terms described in the LICENSE file in
 # the root directory of this source tree.
-
+from llama_models.sku_list import resolve_model
 from together import Together
 
+from llama_models.llama3.api.datatypes import *  # noqa: F403
+from llama_stack.apis.safety import (
+    RunShieldResponse,
+    Safety,
+    SafetyViolation,
+    ViolationLevel,
+)
 from llama_stack.distribution.request_headers import get_request_provider_data
 
-from .config import TogetherProviderDataValidator, TogetherSafetyConfig
+from .config import TogetherSafetyConfig
+
+SAFETY_SHIELD_TYPES = {
+    "Llama-Guard-3-8B": "meta-llama/Meta-Llama-Guard-3-8B",
```

Comment:
does the vision shield (Llama-Guard-3-11B-Vision) also work? i.e., is the model deployed on the backend? If so I will make it work with the rest of the infra on the Stack.

Reply:
vision shield is added here